p2p/lightchain.py

```diff
@@ -45,22 +44,25 @@
     PeerPool,
     PeerPoolSubscriber,
 )
+from p2p.service import (
+    BaseService,
+)
 
 if TYPE_CHECKING:
     from trinity.db.header import BaseAsyncHeaderDB  # noqa: F401
 
 
-class LightPeerChain(PeerPoolSubscriber):
+class LightPeerChain(BaseService, PeerPoolSubscriber):
```

Comment:
*nitpick*

The class ordering on this is backwards from what I expected.  `PeerPoolSubscriber` is a mixin interface and I've normally put these in the front.  Do you have the opposite convention?